src/main/java/com/zufar/icedlatte/order/entity/Order.java

```diff
@@ -41,14 +42,15 @@ public class Order {
     private UUID userId;
 
```

Comment:
Why `Order` entity has no `totalPrice` attribute?

Where is final sum of all products price?